projects/demo/src/modules/charts/line-chart/examples/6/index.html

```diff
@@ -0,0 +1,28 @@
+<tui-axes
+    class="axes"
+    [verticalLines]="4"
+    [horizontalLines]="2"
+>
+    <tui-line-chart
+        [x]="0"
+        [y]="0"
+        [width]="400"
+        [height]="200"
+        [dotContent]="dot"
+        [value]="value"
+        [xStringify]="stringify"
+        [yStringify]="stringify"
+    ></tui-line-chart>
+</tui-axes>
+
+<ng-template
+    #dot
+    let-point
+    let-hovered="hovered"
```

Comment:
Can we use `(tuiHoveredChange)` (on svg) instead ?

Or it does not work ?

Reply:
We need to show icons on hover on entire column, not only on icon, thats why I use hovered from context (it means hovered column)